<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>changePwd</class>
 <widget class="QDialog" name="changePwd">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>483</width>
    <height>366</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>changePwd</string>
  </property>
  <widget class="QLabel" name="pwdLab">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>140</y>
     <width>54</width>
     <height>12</height>
    </rect>
   </property>
   <property name="text">
    <string>密   码：</string>
   </property>
  </widget>
  <widget class="QLabel" name="idLab">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>100</y>
     <width>54</width>
     <height>12</height>
    </rect>
   </property>
   <property name="text">
    <string>编   号：</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="pwdlineEdit">
   <property name="geometry">
    <rect>
     <x>220</x>
     <y>140</y>
     <width>113</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="checkPwdLab">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>180</y>
     <width>54</width>
     <height>12</height>
    </rect>
   </property>
   <property name="text">
    <string>确认密码：</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="idlineEdit">
   <property name="geometry">
    <rect>
     <x>220</x>
     <y>100</y>
     <width>113</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="checkpwdlineEdit">
   <property name="geometry">
    <rect>
     <x>220</x>
     <y>180</y>
     <width>113</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="okBtn">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>250</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>确认</string>
   </property>
  </widget>
  <widget class="QPushButton" name="cancelBtn">
   <property name="geometry">
    <rect>
     <x>280</x>
     <y>250</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>取消</string>
   </property>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections>
  <connection>
   <sender>cancelBtn</sender>
   <signal>clicked()</signal>
   <receiver>changePwd</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>318</x>
     <y>271</y>
    </hint>
    <hint type="destinationlabel">
     <x>322</x>
     <y>375</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
